File: SEP3_UserSystem/src/main/java/com/example/SEP3_UserSystem/mediator/information/SearchPackage.java
package com.example.SEP3_UserSystem.mediator.information;

public class SearchPackage extends InformationPackage{
    private String Id;

    public SearchPackage(String id,String keyword)
    {
        super(InformationType.SEARCH,keyword);
        Id = id;
    }

    public String getId() {
        return Id;
    }
}

File: SEP3_Client1/Mediator/Information/SearchPackage.cs
namespace SEP3_Client.Mediator.Information
{
    public class SearchPackage : InformationPackage
    {
        public string Id { get; set; }

        public SearchPackage(string id, string keyword) : base(InformationType.SEARCH, keyword)
        {
            Id = id;
        }

        public SearchPackage() {}

        public string GetId()
        {
            return Id;
        }
    }
}